Pearson-Advance/openedx-themes
# PAE-166 - Styling the Profile page on the Regis theme

## Description:
This PR adds the style for the Profile page on the Regis theme

## Screenshots:
![Screenshot 2022-07-21 at 18-23-27 Learner Profile Your Platform Name Here](https://user-images.githubusercontent.com/62396424/180573333-09ba4576-5b33-49d9-9eed-8addc9f5e612.png)
![Screenshot 2022-07-21 at 18-38-29 Learner Profile Your Platform Name Here](https://user-images.githubusercontent.com/62396424/180573379-eb428c28-1943-4c42-becb-bc66aa22cd54.png)

All screenshots: https://imgur.com/a/hK17KZa

## Testing:

- Enable badges to display achievements tab, https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/enable_badging.html
- Check out to this branch.
- Enable Comprehensive theming: https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/configuration/changing_appearance/theming/enable_themes.html
- Compile assets, `paver update_assets --themes=pearson-regis-theme`
- Assign 'pearson-regis-theme' to a new site.

## Reviewers:
